doc/source/whatsnew/v0.20.0.txt

```diff
@@ -1011,6 +1011,7 @@ I/O
 - Bug in ``DataFrame.to_stata()`` and ``StataWriter`` which produces incorrectly formatted files to be produced for some locales (:issue:`13856`)
 - Bug in ``StataReader`` and ``StataWriter`` which allows invalid encodings (:issue:`15723`)
 - Bug in ``pd.to_json()`` for the C engine where rollover was not correctly handled for case where frac is odd and diff is exactly 0.5 (:issue:`15716`, :issue:`15864`)
+- Bug in ``pd.read_csv()`` when an index was specified and no values were specified as null values (:issue:`15835`)
```

Comment:
Based on your comment <a href="https://github.com/pandas-dev/pandas/pull/15881#issuecomment-291227795">below</a>, I'll leave this alone.